proposals/0107-unsaferawpointer.md

```diff
@@ -1587,12 +1589,14 @@ extension UnsafeMutablePointer<Pointee> {
   func deallocate(capacity: Int)
 
   // Naming conventions.
```

Comment:
This is a list of methods that only changed with respect to their naming conventions. Next time I'll send a PR on the branch that defines UnsafeRawPointer for more thorough review.